### ISSUE TITLE
Stack Overflow are the funniest errors.

### ISSUE THREAD

**Issue by LuxusDarkangel** [hilburn/AdvancedSystemsManager#72]
Yes, i'm trying to not use the same title everytime, sorry.
First of all, i'm running AdvancedSystemsManager on my 35+ ppl server since 0.1.13, so you can consider me your biggest feedback source, yay.
Some minutes ago it shown a crash on console (which i will post right down) about ASM, buuut as you can see in the log, the server continued running normally, at 19 tps.
Here you have the pastebin link: http://pastebin.com/V2GDRefA

**Comment by hilburn** [comment]
By any chance did someone make a variable which contained itself? I think that's a feasible source of this bug

**Comment by LuxusDarkangel** [comment]
If you mean creating a variable which contains the ASM where the variable has been created, that's not he problem. If that's not what you mean, please, explain it a bit better.

**Comment by hilburn** [comment]
I mean, when you declare variables and you add a ton of blocks to it, you can also add other variables if it is appropriate. I can see a situation where it allows you to add eg Red to the Red variable which would create this stack overflow

Regardless, I need to know what they assigned to the variable to be able to debug the issue.

**Comment by LuxusDarkangel** [comment]
I know before this some users were trying to use ASM to automate the production of botania's livingwood using block gates, but i'm trying a lot of things and i can't reproduce this.

**Comment by hilburn** [comment]
Any chance you've had this occur since or have a better idea of what caused it? I have been unable to replicate it

**Comment by LuxusDarkangel** [comment]
Yeah, i've seen this more times, but i'm unable to replicate it too. If I can help you in any way...
